packages/keyring-utils/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.0]
+
+### Added
+
+- Add `Keyring` and `KeyringClass` types ([#201](https://github.com/MetaMask/accounts/pull/201)) ([#205](https://github.com/MetaMask/accounts/pull/205)) 
+  - The two types have been migrated from `@metamask/utils`
```

Comment:
Cosmetic:

```suggestion
  - The two types have been migrated from `@metamask/utils`.
```

Reply:
Yea that's true, I omitted it initially because this is transparent for a consumer of this package so it would be confusing to place it under `Changed`. Perhaps a new line under this one that is already present under `Add`?

Reply:
Looks good now, thanks :)